tests/dra-native/dra_native_suite_test.go

```
@@ -0,0 +1,46 @@
+package dranative
```

golangci-lint (2.13.2)

[warning] filename-format: Filename dra_native_suite_test.go is not of the format ^[a-z][a-z0-9]*(_suite_test|_test|_linux|_darwin|_windows|_amd64|_arm64)?\.go$.

(revive)